go-controller/pkg/ovn/logical_switch_manager/logical_switch_manager.go

```diff
@@ -354,6 +367,12 @@ func NewJoinLogicalSwitchIPManager(nbClient libovsdbclient.Client, existingNodeN
 	return &j, nil
 }
 
+//GetUUID returns the UUID of the join switch
+func (jsIPManager *JoinSwitchIPManager) GetUUID() string {
```

Comment:
@JacobTanenbaum I don't think this method gets used anywhere? If not, let's drop it.

Reply:
done